src/main/java/org/gridsuite/study/server/NetworkModificationTreeService.java

```diff
@@ -519,16 +526,17 @@ public void invalidateBuild(UUID nodeUuid, boolean invalidateOnlyChildrenBuildSt
         nodesRepository.findById(nodeUuid).ifPresent(n -> {
             // No need to invalidate a node with a status different of "BUILT"
             if (repositories.get(n.getType()).getBuildStatus(n.getIdNode()) == BuildStatus.BUILT) {
+                fillInvalidateNodeInfos(n, invalidateNodeInfos, invalidateOnlyChildrenBuildStatus);
```

Comment:
Is the move of this line linked with the scope of this PR?
Seems like it could be a fix of another problem no ?

Reply:
Yes
otherwise we lose the starting ids